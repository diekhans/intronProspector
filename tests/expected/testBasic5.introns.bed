chr10	5098879	5099326	sj0	1	.	5098879	5099326	64,128,64
chr10	5453560	5454264	sj1	41	.	5453560	5454264	64,128,64
chr10	5454522	5454947	sj2	53	.	5454522	5454947	64,128,64
chr10	5455118	5456086	sj3	45	.	5455118	5456086	64,128,64
chr10	5456273	5456587	sj4	46	.	5456273	5456587	64,128,64
