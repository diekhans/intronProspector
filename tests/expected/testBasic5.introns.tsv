chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang	confidence	splice_sites	name
chr10	5098879	5099326	.	1	0	0	80	17	0		sj0
chr10	5453560	5454264	.	41	0	0	68	86	4.03		sj1
chr10	5454522	5454947	.	53	0	0	87	89	4.39		sj2
chr10	5455118	5456086	.	45	0	0	85	90	4.75		sj3
chr10	5456273	5456587	.	46	0	0	86	79	4.69		sj4
